### DISCUSSION TITLE
Taille Liste des acteurs

### DISCUSSION THREAD

**Discussion by WITNicolas** [witsa/synapps#663] [Ideas]
Pour pinailler un peu :) Je le redis encore, quand on parle de détails, c'est que le reste est top
Pour les novices le menu est parfait. Icones + Texte en gros
Quand on commence à prendre l'habitude finalement on ne regarde plus que les icones voire l'icone + texte mais en ligne.

Bref, un rangement à la windows serait il envisageable ? Genre dans les paramétres par défaut de STUDIO

Histoire de pouvoir avoir un menu hyper compact pour ceux qui préférent.
Finalement dans les IDE, c'est tout petit en fait et quasi qu'avec des icones
![image](https://github.com/witsa/synapps/assets/33632664/fca05afe-e8dd-4592-ac88-a6400e180e69)

Du coup ça serait plus lisible en liste pour les composites par exemples, vu qu'ils ont par défaut tous la même icone, les textes ont leur importance.
Au départ nous avons 1 ou 2 composites, mais à la longue et l'experience, la liste s'allonge fortement et trés vite.

La Sidebar quoi ...
Toute inspiration est totalement ...volontaire ... 🤣 
![image](https://github.com/witsa/synapps/assets/33632664/e1eac29b-d1c8-4bc6-944d-442b2c2f235a)

**Comment by manufitoussiwit** [comment]
C'est une bonne idée! Du coup, un truc qui devrait te plaire avec la révision d'aujourd'hui : 

[📄 Filtre de recherche de type d’acteur à ajouter](https://witsa.github.io/synapps/synapps-studio-releases/notes/1.2.0#filtre-de-recherche-de-type-dacteur-%C3%A0-ajouter)

![image](https://witsa.github.io/synapps/assets/notes/1.2.0/new-actor-filter.gif)

C'est pas tout à fait ce que tu propose mais pour aller chercher un composite c'est pas mal :-)

**Reply by WITNicolas** [reply]
Oui j'ai vu c'est pas mal du tout ça !
